pkg/controller/feed/reconcile.go

```diff
@@ -101,10 +112,6 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	// to do this properly.
 	// TODO: Add issue link here. can't look up right now, no wifi
 	r.setEventTypeOwnerReference(feed)
-	if err := r.updateOwnerReferences(feed); err != nil {
-		glog.Errorf("failed to update Feed owner references: %v", err)
-		return reconcile.Result{}, err
-	}
 
 	if feed.GetDeletionTimestamp() == nil {
```

Comment:
It looks like this starts jobs that will act on the feed. Because we have delayed the updateFeed call until after this, is there a chance we are introducing a race condition (albeit a rare one, because this function and its caller should return long, long before the job is scheduled and actually starts running)?

Reply:
I don't think there's a race condition because the Job and its children don't depend on the status of the feed (currently). The Job pod gets all the information it needs through env vars.

If we later come up with a feedlet that depends on Feed status existing before the feedlet starts, then we may have to revisit.